src/audits/performance/input-readiness-metric.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+
+/* global window */
+window.global = window;
+
+// we need gl-matrix and jszip for traceviewer
+// since it has internal forks for isNode and they get mixed up during
+// browserify, we require them locally here and global-ize them.
+// from catapult/tracing/tracing/base/math.html
+const glMatrixModule = require('gl-matrix');
+Object.keys(glMatrixModule).forEach(exportName => {
+  global[exportName] = glMatrixModule[exportName];
+});
+// from catapult/tracing/tracing/extras/importer/jszip.html
+global.JSZip = require('jszip/dist/jszip.min.js');
+
+global.HTMLImportsLoader = {};
+global.HTMLImportsLoader.hrefToAbsolutePath = function(path) {
+  if (path === '/gl-matrix-min.js') {
+    return 'empty-module';
+  }
+  if (path === '/jszip.min.js') {
+    return 'jszip/dist/jszip.min.js';
+  }
+};
+
+require('../../../third_party/traceviewer-js/');
+const traceviewer = global.tr;
+
+class InputReadinessMetric extends Audit {
+  /**
+   * @override
+   */
+  static get tags() {
```

Comment:
need to update to reflect #312. see e.g. [src/audits/html/meta-theme-color.js](https://github.com/GoogleChrome/lighthouse/blob/7fb2ecb922308fe5dd06af72b4a95bf5f6c6eac5/src/audits/html/meta-theme-color.js)